config/i18n/locales/swahili/translations.json

```diff
@@ -1,67 +1,67 @@
 {
   "buttons": {
-    "forum": "Forum",
-    "donate": "Donate",
-    "load-more-articles": "Load More Articles"
+    "forum": "Jukwaa",
+    "donate": "Changia",
+    "load-more-articles": "Onyesha Makala Zaidi"
   },
   "search": {
-    "label": "Search",
-    "accessible-name": "Submit your search query",
-    "placeholder": "Search 8,000+ tutorials",
-    "see-results": "See all results for {{ searchQuery }}",
-    "no-tutorials": "No tutorials found"
+    "label": "Tafuta",
+    "accessible-name": "Wasilisha swali lako la utafutaji",
+    "placeholder": "Tafuta kati ya mafunzo zaidi ya 8,000+",
+    "see-results": "Tazama matokeo yote ya {{ searchQuery }}",
+    "no-tutorials": "Hakuna mafunzo yaliyopatikana"
   },
   "banner": {
     "default": "Learn to code — {{ <0> }}free 3,000-hour curriculum{{ </0> }}",
```

Comment:
This string is more noticeable because it's on the banner for people who haven't signed in to freeCodeCamp:

![image](https://user-images.githubusercontent.com/2051070/182130036-f41bb9e1-00cf-42ae-8930-c335442bbb78.png)

But because the Swahili localization for Learn isn't available yet, it would probably be better to link to the [Swahili chatroom on Discord](https://discord.com/channels/692816967895220344/986667820928602162) instead.

Could you adapt this string into Swahili instead?: "Chat with other developers in Swahili"

Please feel free to change the wording if there's a more natural phrase.

Reply:
> 

How do I embed the link to the discord chatroom?